packages/server/modules/core/tests/integration/create-user.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { assert, expect } from 'chai'
+import { createUser, getUser, getUserById } from '../../services/users'
+import { beforeEach, describe, it } from 'mocha'
+import { beforeEachContext } from '@/test/hooks'
+import { UserRecord } from '../../helpers/types'
+import knexInstance from '@/db/knex'
+import { createRandomEmail, createRandomPassword } from '../../helpers/test-helpers'
+import { USER_EMAILS_TABLE_NAME } from '@/modules/user-emails/constants'
+
+const db = knexInstance
+const userEmailsDB = db(USER_EMAILS_TABLE_NAME)
+
+describe('Users @core-users', () => {
+  beforeEach(async () => {
+    await beforeEachContext()
+  })
+
+  it('Should create a user', async () => {
+    const newUser = {
+      name: 'John Doe',
+      email: createRandomEmail(),
+      password: createRandomPassword()
+    }
+
+    const actorId = await createUser(newUser)
+
+    expect(actorId).to.be.a('string')
+  })
+
+  it('Should store user email lowercase', async () => {
+    const user = {
+      name: 'Marty McFly',
+      email: createRandomEmail(),
+      password: createRandomPassword()
+    }
+
+    const userId = await createUser(user)
+
+    const storedUser = await getUser(userId)
+    expect(storedUser.email).to.equal(user.email.toLowerCase())
+  })
+  it('Should not create a user with a too small password', async () => {
+    try {
+      await createUser({
+        name: 'Dim Sum',
+        email: createRandomEmail(),
+        password: createRandomPassword(5)
+      })
+    } catch {
+      return
+    }
+    assert.fail('short pwd')
```

Comment:
1. expectToThrow()
2. You should check what the error actually is, instead of accepting all errors as an OK result